test_4_filter_numbers.py

```diff
@@ -1,6 +1,12 @@
 def filter_numbers(arr):
   # Remove the `return None` code below and add your own code.
-  return None
+  x=0
+  newarr = []
+  while x < len(arr):
```

Comment:
Nice work! This is purely a thought experiment that you can choose whether to take on: How would you approach this problem without a `while` loop?